lib/data/sql/V1.1__import_lu.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE lu_standorttypen (STO_Nr TEXT, STO_DEU TEXT, STO_LAT TEXT, EIGNUNG TEXT, WB_VERJ_ENT TEXT, WB_PFL TEXT, BESCHREIBUNG TEXT, HOEHENVERBREITUNG TEXT, VEGETATION TEXT, VORWALDBAUMARTEN TEXT, GESGR_Nr TEXT, VERDRISK TEXT, PRIORITAET TEXT);
```

Comment:
What do you think about using Postgres schemas for each profile?

Reply:
Maybe at a later time, but we can put it in the backlog for now